src/game/animation/updateObject.ts

```diff
@@ -0,0 +1,9 @@
+export function updateObjectPositionTo(ball: HTMLDivElement, position: { x: number, y: number }): void {
```

Comment:
How about changing the name of 1st arg? :)
```suggestion
export function updateObjectPositionTo(obj: HTMLDivElement, position: { x: number, y: number }): void {
```